src/components/EligibilityForm/components/EligibilityForm.styled.tsx

```diff
@@ -18,28 +18,37 @@ export const CheckEligibilityFormLabel = styled.div<{ centred?: boolean }>`
     `}
 `;
 
-export const ContactFormWrapper = styled.div`
+export const ContactFormWrapper = styled.div<{ cardMode?: boolean }>`
   display: flex;
   flex-direction: column;
   align-items: flex-start;
   justify-content: stretch;
-  padding: 1.5rem 0;
 
-  @media (min-width: 992px) {
-    flex-direction: row;
-  }
+  ${({ cardMode }) => css`
+    ${!cardMode ? 'padding: 1.5rem 0' : ''};
+
+    @media (min-width: 992px) {
+      flex-direction: ${cardMode ? 'column' : 'row'};
+    }
+  `}
 `;
 
-export const ContactFormContentWrapper = styled.div`
+export const ContactFormContentWrapper = styled.div<{
+  cardMode?: boolean;
+}>`
   flex: 1;
 
-  &:first-child {
-    padding-right: 1rem;
-    flex: 0.8;
-  }
-  &:last-child {
-    padding-left: 1rem;
-  }
+  ${({ cardMode }) =>
+    !cardMode &&
+    css`
+      &:first-child {
+        padding-right: 1rem;
+        flex: 0.8;
+      }
+      &:last-child {
+        padding-left: 1rem;
+      }
```

Comment:
the padding right and left should be on the parent

Reply:
It is not a space around the elements, but between the elements.

Reply:
Then you can propably use `gap`